flixel/addons/display/FlxNestedSprite.hx

```diff
@@ -266,14 +266,18 @@ class FlxNestedSprite extends FlxSprite
 					var cos:Float = Math.cos(radians);
 					var sin:Float = Math.sin(radians);
 					
-					var dx:Float = child.relativeX - offset.x;
-					var dy:Float = child.relativeY - offset.y;
+					/* Fix from yannicular */	
```

Comment:
On no, I'm sorry I forgot to remove this line, it was meant to help me while commenting out blocks of code while testing. I really didn't do that for the credit. Should I cancel this request, or post another  commit?
Thanx for your answer